index.js

```diff
@@ -477,3 +598,56 @@ async function handleSkipSong(channel, tags) {
         // ^ TODO get one place to handle common Spotify error codes
     }
 }
+
+async function handleGetVolume(channel, tags) {
+    try {
+        let eligible = isUserEligible(channel, tags, chatbotConfig.volume_set_level);
+
+        if(eligible) {
+            let spotifyHeaders = getSpotifyHeaders();
+            res = await axios.get('https://api.spotify.com/v1/me/player', { headers: spotifyHeaders });
+
+            let currVolume = res.data.device.volume_percent;
+            console.log(`${tags[displayNameTag]}, the current volume is ${currVolume.toString()}!`);
+            client.say(channel, `${tags[displayNameTag]}, the current volume is ${currVolume.toString()}!`);
+        }
+    } catch (error) {
+        console.log(error);
+        // Skipping the error for now, let the users spam it
+        // 403 error of not having premium is the same as with the request,
+        // ^ TODO get one place to handle common Spotify error codes
+    }
+}
+
+async function handleSetVolume(channel, tags, arg) {
+    
+    try {
+        let eligible = isUserEligible(channel, tags, chatbotConfig.volume_set_level);
+
+        if(eligible) {
+
+            let number = 0;
+            try {
+                number = Number(arg);
+                number = clamp(number, volMin, volMax);
+            } catch (error) {
+                console.log(error);
+                client.say(channel, `${tags[displayNameTag]}, a number between 0 and 100 is required.`);
```

Comment:
Can't we just pass in username instead of tags[displayNameTag]?

Reply:
following previously established coding standard.

![image](https://user-images.githubusercontent.com/22343767/201497649-4d02c01e-d4bc-4093-9053-4647545edc0b.png)